packages/manager/apps/web/client/app/domain/anycast/domain-dns-anycast.controller.js

```diff
@@ -1,79 +1,90 @@
+import find from 'lodash/find';
+import get from 'lodash/get';
+
+const DNS_ANYCAST_SERVICE_TYPE = 'dns';
```

Comment:
Maybe those constants could be extracted into their own file `packages/manager/apps/web/client/app/domain/anycast/constants.js`.

It could be easier if we want to reuse them. What do you think?